specification/communication/data-plane/Chat/preview/2022-11-15-preview8/communicationserviceschat.json

```diff
@@ -0,0 +1,1977 @@
+          },
+          "senderDisplayName": {
+            "description": "The display name of the chat message sender. This property is used to populate sender name for push notifications.",
+            "type": "string",
+            "example": "Bob Admin"
+          },
+          "type": {
+            "$ref": "#/definitions/ChatMessageType"
+          },
+          "metadata": {
+            "description": "Message metadata.",
+            "type": "object",
+            "additionalProperties": {
+              "type": "string"
+            }
+          }
+        }
+      },
+      "SendChatMessageResult": {
+        "description": "Result of the send message operation.",
+        "required": [
+          "id"
+        ],
+        "type": "object",
+        "properties": {
+          "id": {
+            "description": "A server-generated message id.",
+            "type": "string",
+            "example": "123456789"
+          }
+        }
+      },
+      "ChatParticipant": {
+        "description": "A participant of the chat thread.",
+        "required": [
+          "communicationIdentifier"
+        ],
+        "type": "object",
+        "properties": {
+          "communicationIdentifier": {
+            "$ref": "#/definitions/CommunicationIdentifierModel"
+          },
+          "displayName": {
+            "description": "Display name for the chat participant.",
+            "type": "string",
+            "example": "Bob"
+          },
+          "shareHistoryTime": {
+            "format": "date-time",
+            "description": "Time from which the chat history is shared with the participant. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`.",
+            "type": "string",
+            "example": "2020-10-30T10:50:50Z"
+          }
+        }
+      },
+      "AttachmentType": {
+        "description": "The type of attachment.",
+        "enum": [
+          "teamsInlineImage"
+        ],
+        "type": "string",
+        "x-ms-enum": {
+          "name": "AttachmentType",
+          "modelAsString": true
+        }
+      },
+      "ChatAttachment": {
+        "description": "An attachment in a chat message.",
+        "type": "object",
+        "properties": {
+          "id": {
+            "description": "Id of the attachment",
+            "type": "string",
+            "example": "A27C7079-626F-4D23-8BEE-582F2C376B5A"
+          },
+          "attachmentType": {
+            "$ref": "#/definitions/AttachmentType"
+          },
+          "contentType": {
+            "description": "The type of content of the attachment as mime type, if available",
+            "type": "string",
+            "example": "text/html"
+          },
+          "name": {
+            "description": "The name of the attachment content.",
+            "type": "string",
+            "example": "filename.txt"
+          },
+          "url": {
+            "description": "The URL where the attachment can be downloaded",
```

Comment:
As discussed offline, let's add a second `previewUrl` so that clients do not need to write teams interop specific logic that would need to be documented outside of the swagger